package/yast2-firewall.spec

```diff
@@ -32,9 +32,13 @@ BuildRequires:  yast2-devtools >= 3.1.10
 BuildRequires:  yast2 >= 4.1.21
 BuildRequires:  rubygem(%rb_default_ruby_abi:yast-rake)
 BuildRequires:  rubygem(%rb_default_ruby_abi:rspec)
+# Yast::UsersSimple.GetRootPassword
+BuildRequires:  yast2-users
 
 # Y2Firewall::Firewalld#reset
 Requires:       yast2 >= 4.1.21
+# Yast::UsersSimple.GetRootPassword
+Requires:       yast2-users
```

Comment:
I'm not sure if we actually want this. It may create some other dependency cycle. At the end, it's not a runtime dependency, just in Installer.